metaflow/client/core.py

```diff
@@ -491,6 +533,21 @@ def tags(self):
         """
         return self._tags
 
+    @property
+    def system_tags(self):
```

Comment:
I don't mind doing it. Would refactor the code a bit though (I would introduce another subclass that would inherit from MetaflowObject and that would be the parent of other objects. Also, would break compatibility (`created_at` change). Am fine doing it though if that is ok. I do agree, it is better.

I am not sure however if it makes sense to do it right now in the sense that tags at the step/task/run/artifact levels are also a bit iffy in terms of what they mean. What is a tag at a step level for example?

I don't have a strong opinion on the matter though.

Reply:
Moved all the tag operations to run; kept `created_at` to avoid breaking changes.